extensions/vscode/webviews/homeView/src/stores/home.ts

```diff
@@ -33,6 +33,7 @@ export const useHomeStore = defineStore("home", () => {
     return credentials.value.sort((a, b) => a.name.localeCompare(b.name));
   });
 
+  const serverSecrets = ref<Set<string>>(new Set());
```

Comment:
I used a Set here to avoid iterating through the array each time, and Environment Variables are unique by name anyway.